phpunit.xml.dist

```diff
@@ -26,4 +31,12 @@
             </exclude>
         </whitelist>
     </filter>
+
+    <logging>
+        <log type="coverage-html" target="./Tests/Logs/coverage.html" lowUpperBound="35"
+             highLowerBound="70"/>
+        <log type="testdox-html" target="./Tests/Logs/Testdox/testdox.html"/>
+        <log type="testdox-text" target="./Tests/Logs/Testdox/testdox.txt"/>
+    </logging>
```

Comment:
I wouldn't include this in this merge request, it is to add another overhead, also the folders where to store this shouldn't be inside the Tests

Reply:
I'll remove this part. I must be commit this by mistake.